api/internal/managers/app/install/install.go

```diff
@@ -97,3 +109,55 @@ func (m *appInstallManager) createConfigValuesFile(configValues kotsv1beta1.Conf
 
 	return configValuesFile.Name(), nil
 }
+
+func (m *appInstallManager) installHelmCharts(ctx context.Context) error {
+	logFn := m.logFn("app-helm")
+
+	if m.releaseData == nil || len(m.releaseData.HelmChartArchives) == 0 {
+		logFn("no helm charts found in release data")
+		return nil
+	}
+
+	logFn("installing %d helm charts from release data", len(m.releaseData.HelmChartArchives))
+
+	for i, chartArchive := range m.releaseData.HelmChartArchives {
+		logFn("installing chart %d/%d", i+1, len(m.releaseData.HelmChartArchives))
+
+		if err := m.installHelmChart(ctx, chartArchive, i); err != nil {
```

Comment:
rather than pass the index here as an argument, can you wrap the error in the index?